pkg/kubectl/cmd/auth/cani_test.go

```diff
@@ -166,6 +174,11 @@ func TestRunAccessCheck(t *testing.T) {
 			}
 
 			actualAllowed, err := test.o.RunAccessCheck()
+			actualReturnCode := 0
+			if !actualAllowed {
+				actualReturnCode = 1
```

Comment:
this isn't really testing your change... this test would pass even without your change. I expected something in test-cmd-util.sh that actually checked exit code

Reply:
You're completely right. I've written an small test in test-cmd-util.sh, but becase the apiserver is always in AlwaysAllow during tests, does it makes sense? It will never return exit 1.

Reply:
hrmm... true. I guess we'll skip that for now and go with manual testing.